packages/ws-client/src/data-source.ts

```diff
@@ -34,7 +34,7 @@ export type UpdateWebsocketMessage = {
 export type EventResponseWebsocketMessage = {
   $: 'evt-res'
   key: string
-  res: ServerResponseDataState
+  res: ResponseDataState
```

Comment:
fix: the type on the client (after json.stringify) is obv. ResponseDataState (object w/o functions)